modules/local/annotate_uniprot.nf

```diff
@@ -0,0 +1,57 @@
+process ANNOTATE_UNIPROT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::bioconductor-uniprot.ws==2.34.0--r41hdfd78af_0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bioconductor-uniprot.ws:2.34.0--r41hdfd78af_0':
+        'bioconductor/bioconductor_docker' }"
```

Comment:
Not sure I understand, but I think I commented above that I can probably just create a docker image that houses the packages I need and pull that one from my dockerhub rather than doing it in this clunky way.

Reply:
Okay, I've made it so that I'm pulling for a docker container that contains everything needed for this module.